extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -118,6 +125,13 @@ public void checkKubernetes(BuildProducer<KubernetesDeploymentTargetBuildItem> d
         }
     }
 
+    @BuildStep
+    public void checkMinishift(BuildProducer<KubernetesDeploymentTargetBuildItem> deploymentTargets) {
```

Comment:
Minikube, not Minishift :-)

Reply:
Oh man.. I don't know how many times I made that mistake...